lib/gruntTasks.js

```diff
@@ -43,12 +43,21 @@ function buildDbMigrateArgs(grunt, command) {
   if (grunt.option('coffee-file')) {
     args.push('--coffee-file')
   }
-  
+
   if (grunt.option('migrations-dir')) {
     args.push('--migrations-dir');
     args.push(grunt.option('migrations-dir'));
   }
 
+  if (grunt.option('table')) {
+    args.push('--table');
+    args.push(grunt.option('table'));
+  }
+  else if (sails.config.migrations.table) {
```

Comment:
This is good thinking, allowing the table to be set in the config. I should probably do that for `coffee-file` and `migrations-dir`, too.